runtime/doc/lua.txt

```diff
@@ -3670,4 +3670,63 @@ totable({f}, {...})                                       *vim.iter.totable()*
     Return: ~
         (table)
 
+
+==============================================================================
+Lua module: vim.snippet                                          *vim.snippet*
+
+active()                                                    *snippet.active()*
+    Returns `true` if there's an active snippet in the current buffer.
+
+    Return: ~
+        (boolean)
```

Comment:
Is it not likely that we'll get a request for a `snippet.get()` in the future, which returns more info than merely a boolean (such as snippet "id" or position, idk)? If so perhaps we should rename this or at least change the return type, else it will be redundant.

Reply:
I can see `snippet.get()` being part of a more mature version of the API that exposes relevant info for external libraries to build on top of. However I'm unsure of what details we'll be exposing.

> If so perhaps we should rename this or at least change the return type, else it will be redundant.

@justinmk I'm not fully understanding the suggestion (and the why) here.

Reply:
currently this just returns whether "something is active". If we instead have a `get()` that returns info about the active thing, then `active()` is redundant.

Reply:
I see. I'm adding this to https://github.com/neovim/neovim/issues/25696 :)